app/Main.hs

```diff
@@ -4,7 +4,7 @@
 module Main where
 
 import Data.Version (showVersion)
-#if MIN_VERSION_optparse_applicative(0,13,0)
+#if !MIN_VERSION_base(4,13,0)
```

Comment:
@fefi95 Why do we need to keep this? Is it because of GHC 8.0? If so, we might want to get rid of that version, too.

Reply:
That version and GHC 8.2, if I remember correctly. I tried to remove it and the CI failed https://github.com/stackbuilders/dotenv-hs/actions/runs/947051275. 

Do you want me to remove support for GHC 8.0 as well?

Reply:
Snoyberg [suggest to support only the last three versions of GHC](https://www.snoyman.com/blog/2018/07/stop-supporting-older-ghcs/) which means 9.0, 8.10, and 8.8. Nevertheless, that's just a rule of thumb is not a strict rule. Therefore, I think it's fine if we support 8.0, 8.2, 8.4 and 8.6 for now. I'd strongly recommend to stop supporting 8.0, and 8.2 at the beginning of next year (Jan 2022). In conclusion, let's not drop support for GHC 8.0 and 8.2, yet.

Reply:
Sure, makes sense! Thanks.